LICENSES/Apache-2.0.txt

```diff
@@ -0,0 +1,73 @@
+Apache License
```

Comment:
Why is this file duplicated? We already have it in the LICENSE file.

Reply:
[Reuse as a tool requires that all licenses are stored in the LICENSES subfolder](https://reuse.software/spec/). 

- `./LICENSE` pretty much indicates the overall license of a project and is used to indicate the license to platforms like GitHub. 
- `./.reuse/dep5` together with `LICENSES` indicates file-specific licenses and copyright information. This is only really relevant if you have "copied" third-party source inside the project e.g. when you vendor dependencies into your project. This spec is used by e.g. https://clearlydefined.io/ to determine all licenses used in a repository.

Ideally you only want one format and the `reuse` approach would be the more precise one, but then you would lose license indication on platform like GitHub. Therefore we opt to have both.

Reply:
Ok